Library/Formula/depqbf.rb

```diff
@@ -2,7 +2,7 @@ class Depqbf < Formula
   homepage "https://lonsing.github.io/depqbf/"
   url "https://github.com/lonsing/depqbf/archive/version-4.0.tar.gz"
   sha1 "dd7dd35aded11bb348ff9ef16626d481b7da3fe4"
-  head "https://github.com/lonsig/depqbf.git"
+  head "https://github.com/lonsing/depqbf.git"
 
   bottle do
     cellar :any
```

Comment:
Ok, no worries. I wonder if it's worth submitting a patch for one? Maybe not, up to you :)